config/initializers/permissions_setter.rb

```diff
@@ -0,0 +1,13 @@
+# Be sure to restart your server when you modify this file.
+
+# You can add backtrace silencers for libraries that you're using but don't wish to see in your backtraces.
+# Rails.backtrace_cleaner.add_silencer { |line| line =~ /my_noisy_library/ }
+
+# You can also remove all the silencers if you're trying to debug a problem that might stem from framework code.
+# Rails.backtrace_cleaner.remove_silencers!
+require 'fileutils'
+
+FileUtils.touch(Configuration.yaml_file_path)
```

Comment:
Before touching files, do

```ruby
File.umask(2)
```

Since all of the admin users of this Rails app will not be the bmiadmin user.

Reply:
Also `FileUtils.chmod(0775 ` should be `FileUtils.chmod(0664`. That was my bad on the issue.